pagination/pagetoken_test.go

```diff
@@ -35,6 +36,52 @@ func TestParseOffsetPageToken(t *testing.T) {
 		assert.Equal(t, int64(30), pageToken3.Offset)
 	})
 
+	t.Run("skip", func(t *testing.T) {
+		request1 := &freightv1.ListSitesRequest{
+			Parent:    "shippers/1",
+			PageSize:  20,
+			PageToken: "",
+			Skip:      30,
+		}
+		pageToken1, err := ParsePageToken(request1)
+		assert.NilError(t, err)
+		assert.Equal(t, int64(0), pageToken1.Offset)
```

Comment:
I don't get this; we request 20 `site` objects and skip the first `30`. Shouldn't the `pageToken1.Offset` be set to `30`?

Reply:
Ah, never mind. I got it now. Looks good to me :)